### DISCUSSION TITLE
WHats the max i can downgrade on kernver5

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1288] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
WHats the max i can downgrade on kernver5?

**Comment by Luigi-Master** [comment]
When I last used this, I could only go 3 versions down. But they probably improved it now, I will check further.

**Comment by Charlieis1nsan3** [comment]
GOD DAMN kernver 5!????   I'm on kernver 4.

**Comment by nathblade16** [comment, marked as answer]
130 is the furthest you can go
